_includes/articles/suredbits-enterprise-ln.md

```diff
@@ -0,0 +1,58 @@
+{:.post-meta}
```

Comment:
I suggest we delay this until next week. Even without this, it's a very long and ligthning-heavy issue

The field report isn't time-sensitive, so we can publish it any time.